exasol/ai/mcp/server/utils.py

```diff
@@ -0,0 +1,12 @@
+import json
+import logging
+
+from mcp.types import TextContent
+
+logger = logging.getLogger("exasol_mcp_server")
+
+
+def report_error(tool_name: str, error_message: str) -> TextContent:
```

Comment:
I am not completely sure if returning an error as text content is the best idea. In general, it seems you can throw exceptions. https://gofastmcp.com/servers/tools#error-handling